consts/regexp.js

```diff
@@ -4,6 +4,7 @@ const uaLanguageRegExp = new RegExp(/^[а-яїієґ0-9\s]+$/i);
 const enLanguageRegExp = new RegExp(/^[a-z0-9\s]+$/i);
 const userNameRegExp = /^(?=.{2,30}$)[a-zA-Zа-яА-ЯІЄЇіїє]+(([',. -][a-zA-Zа-яА-ЯІЄЇіїє])?[a-zA-Zа-яА-ЯІЄЇіїє]*)*$/u;
 const passwordRegExp = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,30}$/;
+const zipcode = /^\d{5}(?:[-\s]\d{4})?$/;
```

Comment:
Please rename 'zipcode' to 'zipcodeRegExp'

Reply:
fixed